bin/run-tc-ldc93s1_checkpoint.sh

```diff
@@ -1,23 +1,31 @@
-#!/bin/sh
+#!/bin/bash
 
-set -xe
+set -xe pipefail
 
 ldc93s1_dir="./data/ldc93s1-tc"
 ldc93s1_csv="${ldc93s1_dir}/ldc93s1.csv"
 
+epoch_count=$1
+
 if [ ! -f "${ldc93s1_dir}/ldc93s1.csv" ]; then
     echo "Downloading and preprocessing LDC93S1 example data, saving in ${ldc93s1_dir}."
     python -u bin/import_ldc93s1.py ${ldc93s1_dir}
 fi;
 
-python -u DeepSpeech.py \
+python -u DeepSpeech.py --log_level 3 --noshow_progressbar \
```

Comment:
Maybe `set -o pipefail` should be just before the python call, and immediately followed by a `set +o pipefail` ?